src/osemgrep/configuring/Semgrep_settings.ml

```diff
@@ -0,0 +1,97 @@
+type t = {
+  has_shown_metrics_notification : bool option;
+  api_token : string option;
+  anonymous_user_id : Uuidm.t;
+}
+
+let default_settings =
+  {
+    has_shown_metrics_notification = None;
+    api_token = None;
+    anonymous_user_id = Uuidm.v `V4;
+  }
+
+let _t = ref default_settings
```

Comment:
why the _prefix? make it sound like it's not used.

Reply:
removed in [9306c04](https://github.com/returntocorp/semgrep/pull/7637/commits/9306c04c582e7d6724195fea1e0b102f0dc9335a)